### DISCUSSION TITLE
Win10播放有概率自动切歌？

### DISCUSSION THREAD

**Discussion by ApigeonAPE** [HyPlayer/HyPlayer#138] [Q&A] [answered]
*系统：Windows 10 家庭版 21H2 内部版本19044.1526*

~~系统正常自动锁屏后~~ 现在发现正常播放时也会，**时不时出现一首歌放了个开头就自动切换下一首的问题。**
*软件版本：2.1.14.0*
*设备识别码：aab93f34-ec42-ebde-a035-7ed869c1d819*

实验室中开启了亚克力背景，播放缓存，高级音乐历史，听歌打卡

**Comment by ApigeonAPE** [comment, marked as answer]
发现问题所在了。原来是VIP歌曲试听，所以没法放完。居然不能显示哪些歌是VIP，有点遗憾，下载时VIP歌曲不能下载也不会提醒。希望重构版能加入VIP显示，也免得踩坑了。

**Reply by kengwang** [reply]
感谢反馈, 我把它 Transform to a issue, 这个旧版就要修

**Reply by kengwang** [reply]
顺, 这种问题可以直接发成 Issue, 方便追踪一点
